### ISSUE TITLE
Suggested updates to the Summary

### ISSUE THREAD

**Issue by SteveALee** [w3c/coga#138]
https://www.w3.org/TR/coga-usable/#summary

1. @michael-n-cooper provided some css to improve the look of the lists. See below
2. The last item "Test with real users" actually applies to all the objectives and is not really one itself. It is allso a general a11y principle. Thus, I suggest it is moved out of the list. It can be made a stand alone paragraph at the end if it is felt to add somehting important that is coga specific.
3. We should be consistent with the list of Objectives as they are used throughout the document and provide a usefull high level list of topics to keep in mind for Cognitive Accessibility. In the summary "Objective 3: Use Clear and Understandable Content" has been split into two: "Help users understand with clear text and images" and "Provide support for different ways to understand content". I suggest they are remerged in the Summary or else if the distinction is concidered usefull the are separated in the rest of thee document (I do think there is some merit in that)

Suggested CSS
```
#summary li img {
   float: left;
   margin: 0 0.3em 0.5em -4em;
}
#summary ol {
   margin-left: 2em;
}
#summary ol > li {
   margin-bottom: 1em;
   list-style-type: none;
} 
```

**Comment by julierawe** [comment]
Seconding these comments about avoiding inconsistency. We think it would be helpful for the summary to list the same objectives as the ones listed in the table of contents.

**Comment by lseeman** [comment]
steve, please close

**Comment by jpascalides** [comment]
Thank you for your feedback. We have modified the first sentence of the summary to address these comments.
